tests/auth_tests/test_forms.py

```diff
@@ -778,6 +778,13 @@ def test_username_field_autocapitalize_none(self):
         form = UserChangeForm()
         self.assertEqual(form.fields['username'].widget.attrs.get('autocapitalize'), 'none')
 
+    def test_password_field_disabled(self):
+        form = UserChangeForm()
+        # Ensures that the password field is disabled and is
+        # considered unchanged regardless of the specified values
+        self.assertTrue(form.fields['password'].disabled)
+        self.assertFalse(form.fields['password'].has_changed("x", "y"))
```

Comment:
This is already tested in `ReadOnlyPasswordHashTest.test_readonly_field_has_changed()` so I would only add `self.assertIs(form.fields['password'].disabled, True)` assertion to the `test_readonly_field_has_changed()`.